clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/JsonHubProtocol.java

```diff
@@ -114,9 +119,14 @@ public TransferFormat getTransferFormat() {
                         hubMessages.add(new InvocationMessage(target, arguments.toArray()));
                     }
                     break;
+                case COMPLETION:
+                    if (resultToken != null) {
+                        result = gson.fromJson(resultToken, binder.getReturnType(invocationId));
```

Comment:
Does this parser work if properties are out of order?

Reply:
https://github.com/aspnet/SignalR/pull/2961/files#diff-40b6641977b917a809fd35253a9368ffR218